changelog/12006.misc.md

```diff
@@ -0,0 +1 @@
+Apply BILOU tagging to the message in the DIET classifier only when DIET used as an entity recognizer.
```

Comment:
I decided to give the changelog entry the `misc` type because this PR fixes no reported bug or security vulnerability. 
This PR doesn't change any current behaviour but is necessary for future changes.

Reply:
ok 👍🏻